htsget.md

```diff
@@ -165,7 +165,7 @@ The server SHOULD reply with an `UnsupportedFormat` error if the requested forma
 `referenceName` 
 _optional_
 </td><td>
-The reference sequence name, for example "chr1", "1", or "chrX". If unspecified, all reads (mapped and unmapped) are returned. [^b]
+The reference sequence name, for example "chr1", "1", or "chrX". If "*", unmapped (unplaced) reads are returned. If unspecified, all reads (mapped and unmapped) are returned.
```

Comment:
I think that for correctly formatting in markdown you should scape the `*` (using `\*`)